database/types.go

```diff
@@ -46,3 +46,8 @@ type EthSendRawTxEntry struct {
 	TxData                      string    `db:"tx_data"`
 	TxSmartContractMethod       string    `db:"tx_smart_contract_method"`
 }
+
+type Entry struct {
```

Comment:
`Entry` seems too generic of a name. maybe `EntryContainer` or `RequestEntryContainer` describes a bit better what it does